content/docs/guides/10-minute-tutorial.md

```diff
@@ -897,7 +897,7 @@ package hellocucumber;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.When;
 import io.cucumber.java.en.Then;
-import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
```suggestion
import static org.junit.jupiter.api.Assertions.*;
```

While wildcard imports aren't great for use in production code and usually not needed due to IDE hinting they smooth out tutorials where these are not available.